Example/Tests/Error/DMEErrorTests.swift

```diff
@@ -0,0 +1,24 @@
+//
+//  DMEErrorTests.swift
+//  DigiMeSDKExample_Tests
+//
+//  Created on 23/09/2019.
+//  Copyright © 2019 CocoaPods. All rights reserved.
+//
+
+import XCTest
+
+class DMEErrorTests: XCTestCase {
+
+    func testSDKError() {
+        XCTAssertEqual("me.digi.sdk", SDK_ERROR)
```

Comment:
This is definitely a great start, but perhaps a better test would be to create entire error object, and check the domain is set correctly, for example:

```swift
let error = NSError.sdkError(.noAppId)
XCTAssertEqual("me.digi.sdk", error.domain)
```

This way you check whether expected domain is set when this type of error is invoked, as opposed to Just testing the constant.

The same comment applis to the ones below too :)